CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### Chore & Maintenance
 
 - `[*]` Make sure to include `d.ts` files in the tarball when building ([#8086](https://github.com/facebook/jest/pull/8086))
+- `[@babel/plugin-proposal-class-properties]` Use property initializer syntax in Jest codebase  [#8117](https://github.com/facebook/jest/pull/8117)
```

Comment:
We don't usually mention external packages here, the brackets are for the Jest packages affected. I think just `[*]` would be appropriate in this case.

Reply:
Thanks for the comment! Sounds good to me -- changed it to `[*]` .